### PR TITLE
Disable channel actions

### PR DESCRIPTION
**Pull request by bonomat**
No channel 
image

Channel needs confirmation
image

lightning wallet, no channel
image

can't close position if channel is not available
image

resolves https://github.com/itchysats/10101/issues/485
resolves https://github.com/itchysats/10101/issues/428

### CODE REVIEW

**Review by klochowicz — APPROVED** [review]
nice improvement

File: lib/cfd_trading/cfd_order_detail.dart [hunk 1]
```diff
@@ -170,6 +181,7 @@ class _CfdOrderDetailState extends State<CfdOrderDetail> {
                             Visibility(
                               visible: confirm,
                               child: SubmitButton(
+                                  isButtonDisabled: disableActionButton,
```
Comment (holzeis) [inline lib/cfd_trading/cfd_order_detail.dart:184]: nit: no need for this variable you could simply add `!channel.isAvailable()` here.

File: lib/wallet/wallet_lightning.dart [hunk 2]
```diff
@@ -91,21 +113,24 @@ class WalletLightning extends StatelessWidget {
...
-        visible: !hasOpenCfds,
-        closeManually: false,
-        curve: Curves.bounceIn,
-        overlayColor: Colors.black,
-        overlayOpacity: 0.5,
-        elevation: 8.0,
-        shape: const CircleBorder(),
-        children: dials,
+      floatingActionButton: Visibility(
+        visible: showActionButton,
```
Comment (holzeis) [inline lib/wallet/wallet_lightning.dart:117]: nit: we don't need that variable, you could use something like `channel.isUnavailable()`

**Review by holzeis — APPROVED** [review]
Nice 👍

**Pull request merged**